### PR TITLE
Add d2l-switch component

### PR DESCRIPTION
**Pull request by kieranderson**

### CODE REVIEW

**Comment by ppaskaris-d2l** [comment]
I have no idea how this works. Since my review is invalid (I am not a repo contributor), I would recommend you reach out to someone in UXD or something to help facilitate this.

**Comment by kieranderson** [comment]
+Dave, because Github suggested you :)

File: polymer.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "web-components/d2l-rubric.js",
     "web-components/d2l-rubric-editor.js",
     "web-components/d2l-outcomes-level-of-achievements.js",
-    "web-components/d2l-outcomes-user-progress.js"
+    "web-components/d2l-outcomes-user-progress.js",
+    "web-components/d2l-switch.js"
```
Comment (dbatiste) [inline polymer.json]: My current thinking is that this should be moved to `bsi.js`.  @dlockhart , thoughts?

Comment (dlockhart) [reply polymer.json]: The only reason to make it a separate shard would be if we only want to import it on pages that use the component.

Comment (kieranderson) [reply polymer.json]: That is what I was planning on doing, however in the future we most likely want to include it in all pages.

So I can move it now to `bsi.js` if preferred.

Comment (kieranderson) [reply polymer.json]: Moved into `bsi.js` Thanks.

**Comment by kieranderson** [comment]
Updated Looking to get this in for use in a single Renderer for now. Let me know if this is good.

**Pull request merged**